java/client/src/main/java/glide/managers/ConnectionManager.java

```diff
@@ -149,13 +165,25 @@ private ConnectionRequest.Builder setupConnectionRequestBuilderRedisClient(
      *
      * @param configuration
      */
-    private ConnectionRequestOuterClass.ConnectionRequest.Builder
-            setupConnectionRequestBuilderRedisClusterClient(
-                    RedisClusterClientConfiguration configuration) {
+    private ConnectionRequest.Builder setupConnectionRequestBuilderRedisClusterClient(
+            RedisClusterClientConfiguration configuration) {
         ConnectionRequest.Builder connectionRequestBuilder =
                 setupConnectionRequestBuilderBaseConfiguration(configuration);
         connectionRequestBuilder.setClusterModeEnabled(true);
 
+        if (configuration.getSubscriptionConfiguration() != null) {
```

Comment:
seems like duplicate code

Reply:
Unfortunately I didn't find a wayt to move it to another function. Classes are different for standalone and cluster clients.